client/src/components/brushableHistogram/footer.js

```diff
@@ -61,7 +61,7 @@ const HistogramFooter = React.memo(
           </span>
         </div>
 
-        {logFoldChange && pvalAdj ? (
+        {logFoldChange !== undefined && pvalAdj !== undefined ? (
```

Comment:
This appears equivalent, but wouldn't catch `null` — is there a specific motivation?

Reply:
✅ good catch

Reply:
```suggestion
        {logFoldChange != null && pvalAdj != null ? (
```